src/app/app.component.html

```diff
@@ -2,8 +2,10 @@
   <div class="spinner"></div>
 </div>
 <ion-app>
-  <ion-split-pane>
-    <ion-menu side="end">
+  <ion-split-pane contentId="main-content">
+    <div id="main-content">
+      <ion-menu side="end" contentId="my-content">
```

Comment:
This looks harmless but it also seems like a redundant menu that we need to learn to use.

Reply:
That's fair. I made the change for the sake of progress. But feel free to definitely make note of it to look further into. We can address alongside the the current issues or revisit it when we tighten things up before deployment. What's your preference?

Reply:
See https://ionicframework.com/docs/api/split-pane for the suggested usage by Ionic. I have just made these changes.